web/styles/subscriptions.css

```diff
@@ -784,6 +775,12 @@ h4.user_group_setting_subsection_title {
     }
 }
 
+#settings_page,
+#groups_overlay,
+#subscription_overlay {
+    container: settings_overlay / inline-size;
```

Comment:
Two things here: 1, let's style container names with hyphens (`settings-overlay`), and 2, let's be sure to mention the container name in `@container` queries. That'll save us a ton of pain in the future, so we don't have to hunt through the DOM and CSS to figure out where a container is specified. So, `@container settings-overlay (width > $something)`.

Reply:
Done and done, thanks!